src/wp2hugo/internal/mediacache/media_cache_setup.go

```diff
@@ -20,6 +22,40 @@ func New(cacheDirPath string) MediaCache {
 	return MediaCache{cacheDirPath: cacheDirPath}
 }
 
+func waitOrStop(resp *http.Response) (int, bool) {
+	timeout := 1
+	stop := false
+	switch resp.StatusCode {
+
+	case 200:
```

Comment:
Use constants like `http.StatusOK` for readability

Reply:
Others are defined as well - https://pkg.go.dev/net/http
See `StatusTooManyRequests` and others

Reply:
Done